setup.py

```diff
@@ -8,7 +8,7 @@
       version=__version__,
       description="Eucalyptus Cloud Services and General System Administrative Utilities",
       long_description="Eucalyptus Cloud Services and General System Administrative Utilities",
-      url="https://github.com/nephomaniac/adminapi",
+      url="https://github.com/eucalyptus/adminapi",
       dependency_links = ['https://github.com/nephomaniac/python-midonetclient/tarball/4_3_0_e#egg=midonetclient'],
```

Comment:
This one is still nephomaniac, change too?

I didn't find any other nephomaniac refs in the code, but I did find more schwans:

./README.md:27:    git clone https://github.com/bigschwan/adminapi.git
./cloud_admin/cloudview/remote_calyptos.py:16:                     "git clone https://github.com/bigschwan/adminapi.git; cd adminapi; "
./cloud_admin/services/README.md:376:[SAMPLE FULL HTML TABLE ON GH-PAGES](http://bigschwan.github.io/eutester/cloud_admin/services_status_sample.html)
./cloud_admin/services/README.md:598:[SAMPLE FULL HTML TABLE ON GH-PAGES](http://bigschwan.github.io/eutester/cloud_admin/service_types_sample.html)